src/entities/work-log-record.ts

```diff
@@ -0,0 +1,36 @@
+import {Column, Entity, PrimaryGeneratedColumn} from "typeorm";
+import {MaxLength, Property, Required} from "@tsed/common";
+
+@Entity("work_log_record")
+export class WorkLogRecord {
+
+  @PrimaryGeneratedColumn()
+  @Property()
+  id: number;
+
+  @Column({name: "project_id"})
+  @Property()
+  @Required()
+  projectId: string;
+
+  @Column()
+  @MaxLength(200)
```

Comment:
Why should we limit to just 200 chars? Make it text and give them opportunity to be creative :D